overlord/devicestate/devicemgr.go

```diff
@@ -169,10 +170,52 @@ func (m *DeviceManager) ensureOperational() error {
 	return nil
 }
 
+// ensureSnaps makes sure that the snaps from seed.yaml get installed
+// with the matching assertions
+func (m *DeviceManager) ensureSeedYaml() error {
+	m.state.Lock()
+	defer m.state.Unlock()
+
+	// FIXME: enable on classic?
```

Comment:
Yeah, let's enable it, but let's do that later as we need to review our model assertions and make sure it's all hanging together for old installations.